src/ape/managers/query.py

```diff
@@ -46,6 +46,11 @@ def perform_block_query(self, query: BlockQuery) -> pd.DataFrame:
         block_dicts_iter = map(partial(get_columns_from_item, query), blocks_iter)
         return pd.DataFrame(columns=query.columns, data=block_dicts_iter)
 
+    @perform_query.register
+    def perform_transaction_query(self, query: BlockTransactionQuery) -> pd.DataFrame:
+        transactions_iter = self.provider.get_transaction(query.block_id)
```

Comment:
This won't work, needs a new method `ProviderAPI.get_transactions_by_block(block_id: HexBytes) -> Iterator[TransactionAPI]`

Reply:
Yeah, I knew this wouldn't work, really was focusing on the rest of this breaking before I came back to this